includes/class-comment.php

```diff
@@ -45,6 +45,11 @@ public static function init() {
 	 */
 	public static function comment_reply_link( $link, $args, $comment ) {
 		if ( self::are_comments_allowed( $comment ) ) {
+			$user_id = get_current_user_id();
+			if ( $user_id && self::was_received( $comment ) && \user_can( $user_id, 'publish_posts' ) ) {
```

Comment:
use `activitypub` cap instead of `publish_posts`

```suggestion
			if ( $user_id && self::was_received( $comment ) && \user_can( $user_id, 'activitypub' ) ) {
```